### ISSUE TITLE
`alt + tab` and task view DO NOT work on latest win11

### ISSUE THREAD

**Issue by l1teng** [valinet/ExplorerPatcher#1847]
Hi, thanks for this great work helps me redesign my windows task bar and bring the legacy win10 style features back to current workspace.

I noticed after the latest windows 11 update, my `alt + tab` key binding did NOT work, neither the task view when click the badge or use the `win + tab` shortcut.

Further more, there are a few information for your analysis.

- as a updated to the latest `ExplorerPatcher`, the `alt + tab` still did not work
- after I removed the `ExplorerPatcher`, the `alt + tab` worked well

Some information about my workspace settings,

- current os version: windows 11 EDU 22H2 22621.2134 (latest@2023-08-14, virtualization behind the PVE)
- Machine: Intel i5-8400 CPU, w/z 16 GB mem

**Comment by githubbrowser2** [comment]
I personally reinstalled explorerpatcher, after that my alt+tab didn't work either, I fixed it by simply **restarting file explorer**.

**Comment by l1teng** [comment]
> I personally reinstalled explorerpatcher, after that my alt+tab didn't work either, I fixed it by simply **restarting file explorer**.

Test again on my PC:

I followed the steps. Uninstalling the `explorerpatcher` first, then restarted my PC and installed the `explorerpatcher` again.
Actually, the `alt + tab` worked well at the login session moment. But it failed to work when I opened the `Windows File Explorer`. There was a sudden restart of `File Explorer` process (I doubt, as there was always a sudden display refresh every time when I restart this process). It got worked finally as I removed the `explorerpather` program.
